### ISSUE TITLE
CMDPHP PHP ERROR WARNING Backtrace: (/graph_xport.php)

### ISSUE THREAD

**Issue by zuka1337** [Cacti/cacti#5217]
# Describe the bug

Cacti Log Backtrace:

## To Reproduce

Has been here since the upgrade:
![image](https://user-images.githubusercontent.com/57998098/216082582-7435fc8b-afa6-4aed-b6ff-d8c2ccc92aab.png)

> 2023/02/01 15:10:51 - CMDPHP PHP ERROR WARNING Backtrace: (/graph_xport.php[118]:CactiErrorHandler())
> 2023/02/01 15:10:51 - CMDPHP PHP ERROR WARNING Backtrace: (/graph_xport.php[95]:CactiErrorHandler())

## Code
graph_xport.php
```
 /* Make graph title the suggested file name */
 95 if (is_array($xport_array['meta'])) {
 96         $filename = $xport_array['meta']['title_cache'] . '.csv';
 97 } else {
 98         $filename = 'graph_export.csv';
 99 }

```
```
 /* Make graph title the suggested file name */
 95 if (is_array($xport_array['meta'])) {
 96         $filename = $xport_array['meta']['title_cache'] . '.csv';
 97 } else {
 98         $filename = 'graph_export.csv';
 99 }
 (...)
```
Cacti Version: 1.2.23
PHP: PHP 5.4.16
OS: Centos7

Upgrade yesterday from 1.2.7 -> 1.2.23

**Comment by TheWitness** [comment]
Suggest you pull 1.2.x branch.  Pretty sure this is fixed there.

**Comment by zuka1337** [comment]
I will try it !

**Comment by netniV** [comment]
You seem to have posted a few duplicate issues.  Hopefully, most of these are going to be addressed by 1.2.24

**Comment by TheWitness** [comment]
Closing for now.  Pretty sure this is all addressed in 1.2.24.
